src/main/java/org/alfresco/repo/content/transform/LocalPassThroughTransform.java

```diff
@@ -76,10 +76,10 @@ public static Transformer getConfig(List<String> mimetypes)
         Set<SupportedSourceAndTarget> supportedSourceAndTargetList = new HashSet();
         for (String mimetype: mimetypes)
         {
-            supportedSourceAndTargetList.add(new SupportedSourceAndTarget(mimetype, mimetype, -1));
+            supportedSourceAndTargetList.add(new SupportedSourceAndTarget(mimetype, mimetype, -1, 20));
```

Comment:
Bug Fix: Legacy pass through had a priority of 20. Without this we would sometimes use more complex/costly transforms.